frontend/layouts/discover-page/component.tsx

```diff
@@ -120,25 +120,28 @@ export const DiscoverPageLayout: FC<DiscoverPageLayoutProps> = ({
 
   return (
     <div className="fixed top-0 bottom-0 left-0 right-0 h-screen bg-background-dark">
-      <div className="flex flex-col h-screen overflow-auto">
+      <div className="flex flex-col h-screen sm:overflow-auto bg-gradient-to-t from-background-green-dark to-background-green-dark bg-[length:100%_166px] bg-no-repeat sm:bg-none">
         <div className="z-10 h-min">
           <Header />
-          <LayoutContainer className="z-10 flex justify-center pt-1 mt-2 mb-2 pointer-events-none xl:pb-0 xl:mb-0 xl:-mt-10 xl:left-0 xl:right-0 xl:relative">
+          <LayoutContainer className="z-10 flex justify-center mt-1 pointer-events-none sm:pt-1 sm:mb-2 xl:pb-0 xl:mb-0 xl:-mt-10 xl:left-0 xl:right-0 xl:relative">
             <DiscoverSearch className="w-full max-w-3xl pointer-events-auto -z-10" />
           </LayoutContainer>
         </div>
-        <main ref={mainRef} className="z-0 flex flex-col flex-grow h-screen overflow-y-auto">
+        <main
+          ref={mainRef}
+          className="z-0 flex flex-col flex-grow h-[calc(100%-180px)] sm:h-screen sm:overflow-y-auto"
+        >
           <LayoutContainer className="">
-            <div className="relative flex flex-col items-center gap-2 pb-2 mx-2 mt-4 mb-6 -ml-1 -mr-1 after:left-2 after:right-2 after:h-px after:bg-[#D3CDC4] after:bg-opacity-40 after:absolute after:-bottom-2 after:lg:bottom-2 lg:mb-1 lg:mt-2 lg:gap-6 lg:flex-row space-between lg:pb-0">
+            <div className="relative flex flex-wrap justify-between gap-2 pb-2 mt-1 mb-2 sm:items-center sm:mt-4 sm:mb-6 sm:-ml-1 sm:-mr-1 md:mb-1 md:mt-2 lg:gap-6 md:pb-0">
```

Comment:
I removed it because the designs just show it on one place, and the presentation doesn't have it anymore.
![Screenshot 2022-10-28 at 16 05 11](https://user-images.githubusercontent.com/48164343/198632036-02a6755f-849a-4c0e-a877-2e82b111f5f2.png)

![Screenshot 2022-10-28 at 16 04 04](https://user-images.githubusercontent.com/48164343/198631825-5bf74f31-ecb3-4ab1-b98d-a5ed094ca077.png)

Reply:
> There used to be a grey border at the bottom of the tabs and sorting options on desktop. Has it been removed because Andreia told you so?
> 
> If you bring it back, check with Andreia when we want to display it. Its position was weird before on mid-size screens.
> 
> I think this is what is causing the 2px difference of the green header background.

I removed it because, on the designs it was removed from all the pages (and the presentation) but the first one.

![Screenshot 2022-10-28 at 16 04 04](https://user-images.githubusercontent.com/48164343/198633818-a2f2cd18-7ec5-4734-b8ab-394f89e6e327.png)
![Screenshot 2022-10-28 at 16 05 11](https://user-images.githubusercontent.com/48164343/198633828-809da4bf-6b4e-4b90-81c8-0718575e4af2.png)